content/post/But_I_only_use_NONMEM.Rmd

```diff
@@ -0,0 +1,142 @@
+"validated" and accepted by regulatory agencies. See 
+[here](http://www.slideshare.net/MikeKSmith/r-eliminating-fear-uncertainty-and-doubt)
+for more discussion on this topic. But if we follow that argument through,
+why aren't people insisting that we re-run NONMEM analyses using PROC NLMIXED?*
+`</end DIGRESSION>`
+
+### I share via publications
+Writing down the maths of a model is a great way to share the model between
+quantitative disciplines. It's a universal language and brings with it rigour
+and a lack of ambiguity. ***BUT*** while two modellers can agree on the maths 
+of a problem, the implementation may vary between two software tools such that
+it may be easy to implement in one tool, but very difficult (impossible?) in 
+another.
+
+## NONMEM doesn't do ***EVERYTHING***
+NONMEM and associated tools e.g. PsN have some limitations in available 
+functionality, for example optimal trial design. And while NONMEM ***can*** be
+used for prediction and simulation of new designs, regimens, scenarios there 
+are a growing number of freely available, open-source tools that can also do 
+this and arguably more easily ***given that the model is encoded in the 
+appropriate input format for that tool***.
+
+Now, either we have to write a load of NMTRAN to <tool X> translators (some 
+exist e.g. Pirana) or we have to use or develop a master language that can 
+easily translate to NMTRAN, but also to other tools e.g. Certara / Pharsight's 
+PML or DDMoRe's MDL.
+
+## Open-source tools are growing in use
+There is a rising number of free, open-source software for nonlinear mixed 
+effects modelling and simulation (e.g. 
+[saemix](http://www.saemix.biostat.fr/), 
+[nlmixr](https://github.com/nlmixrdevelopment/nlmixr), 
+[mlxR / simulx](http://simulx.webpopix.org/), 
+[mrgsolve](http://metrumrg.com/opensourcetools.html), 
+[RxODE](https://github.com/hallowkm/RxODE), 
+[PKPDsim](https://github.com/ronkeizer/PKPDsim), 
+[PFIM](http://www.pfim.biostat.fr/), 
+[PopED](http://poped.sourceforge.net/)
+) and while use of these tools may be limited today, 
+they provide a means for performing the majority of modelling and simulation 
+tasks with no cost to the analyst. More integration of these tools into 
+pharmacometrics toolset and workflow would benefit many, especially those who 
+cannot get access to NONMEM licenses, yet still may wish to use disease and 
+therapeutic drug models in their work or for predicting outcomes for patients.
+
+About 10 years ago, it would have been hard to imagine the case that S-Plus
+would be thoroughly supplanted by R, to the point where it ceased to be a 
+supported product by the company (Tibco) that bought it from Insightful. And yet, here
+we are. [R is rising in popularity as an analytical engine (often coupled with 
+other technologies)](http://blog.revolutionanalytics.com/popularity/). So to 
+dismiss open-source solutions as "irrelevant" may be regarded as a 
+short-sighted view.
+
```

Comment:
Great point @kylebmetrum ! To that point, my response would be the following -
Other than a few folks like yourself and Ron etc, people have not even figured out the possibilities. As a community, if we work together to bring tools into the open and work more collaboratively, and actually produce material with substance, then we can worry about how the regulators or others would accept. 
It comes down to pushing through these hurdles to show that good work can be done collaboratively